src/components/DropdownMenuItems/DropdownMenuItems.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable */
```

Comment:
```suggestion
```

Reply:
Added it back, the reason i removed was due to conflicts with prettier styling, I have updated and adjusted the eslintrc so that we do not run into conflicts with prettier. Now there are no eslint errors.